cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -435,6 +435,31 @@ spec:
     singular: cephblockpool
   scope: Namespaced
   version: v1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            failureDomain:
+                type: string
+            replicated:
+              properties:
+                size:
+                  type: integer
+                  minimum: 0
```

Comment:
I know I suggested to set this min to 0 since it's optional, but now I'm thinking it could be 1 since the parent `size` element is optional. Can you do a quick manual test to verify that if the min is 1, the CRD still passes validation when the EC settings are specified and the replicated.size is removed?